app/main.py

```diff
@@ -16,4 +16,21 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+variables = [lucky_number, pi, one_is_a_prime_number,
+             name, my_favourite_films, profile_info,
+             marks, collection_of_coins
+             ]
+immutable_var = (int, float, str, bool, tuple)
+immutable_list = []
+mutable_list = []
+for value in variables:
+    if type(value) in immutable_var:
+        immutable_list.append(value)
+    else:
+        mutable_list.append(value)
+sorted_variables = {
+    "mutable": mutable_list,
+    "immutable": immutable_list
+}
+
+print(sorted_variables)
```

Comment:
Great  solution, but you made things a bit complicated, you need to write just like this:
```python
sorted_variables = {
    "mutable": [my_favourite_films, ....],
    "immutable": [lucky_number, ....]
}
```
<img width="722" alt="image" src="https://github.com/mate-academy/py-mutable-immutable/assets/93856423/dd7e1dfa-e1f2-4aad-9d34-6b06d86c3190">